### PR TITLE
Auto create monorepo name convention

### PR DESCRIPTION
**Pull request by avishayhajbi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

creating monorepo app files name without context to the app name.
e.g. `app.module.ts ` instead of `my-app-name.module.ts`, `AppModule ` class name instead of `MyAppNameModule`

Issue Number: #262 

## What is the new behavior?
naming monorepo app follows app name
e.g.`my-app-name.module.ts` , `MyAppNameModule`

## Does this PR introduce a breaking change?
```
[ ] Yes
[ x] No
```

## Other information

### CODE REVIEW

File: src/lib/sub-app/schema.json [hunk 1]
```diff
@@ -7,11 +7,7 @@
     "name": {
       "type": "string",
       "description": "The name of the application.",
-      "$default": {
-        "$source": "argv",
-        "index": 0
-      },
-      "x-prompt": "What name would you like to use for the app?"
+      "default": "App"
```
Comment (kamilmysliwiec) [inline src/lib/sub-app/schema.json]: Why this has been changed?

Comment (avishayhajbi) [reply src/lib/sub-app/schema.json]: You right.
It was fixed in the last commit.

Comment (kamilmysliwiec) [reply src/lib/sub-app/schema.json]: It's still not fixed

Comment (avishayhajbi) [reply src/lib/sub-app/schema.json]: You right, I didn't push it. you can see it now :)

**Pull request merged**